src/awesome/screen.rs

```diff
@@ -80,9 +80,12 @@ impl <'lua> Screen<'lua> {
         Ok(Screen::allocate(lua, class)?.build())
     }
 
-    fn init_screens(&mut self, outputs: Vec<Output>) -> rlua::Result<()> {
+    fn init_screens(&mut self, &output: &WlcOutput, outputs: Vec<Output>) -> rlua::Result<()> {
```

Comment:
`WlcOutput` is a `Copy` type (since it's just internally a `uintptr_t`), so passing it by reference is unnecessary here (the borrow checker will be more ok with that, it's more ergonomic to use, and it will probably generate better code because it will be a layer less of indirection).

Reply:
Okay, will do a force push adding the following diff:
```diff
diff --git a/src/awesome/screen.rs b/src/awesome/screen.rs
index 809495e..d7b5751 100644
--- a/src/awesome/screen.rs
+++ b/src/awesome/screen.rs
@@ -80,7 +80,7 @@ impl <'lua> Screen<'lua> {
         Ok(Screen::allocate(lua, class)?.build())
     }
 
-    fn init_screens(&mut self, &output: &WlcOutput, outputs: Vec<Output>) -> rlua::Result<()> {
+    fn init_screens(&mut self, output: WlcOutput, outputs: Vec<Output>) -> rlua::Result<()> {
         let mut state = self.state()?;
         let resolution = output.get_resolution().unwrap();
         state.outputs = outputs;
@@ -123,7 +123,7 @@ pub fn init(lua: &Lua) -> rlua::Result<Class> {
     let mut screens: Vec<Screen> = vec![];
     for output in WlcOutput::list() {
         let mut screen = Screen::cast(Screen::new(lua)?)?;
-        screen.init_screens(&output, vec![output.into()])?;
+        screen.init_screens(output, vec![output.into()])?;
         // TODO Move to Screen impl like the others
         screens.push(screen);
     }
```